conpr/08_LE_Tasks/src/le/fj/Sum.java

```diff
@@ -2,10 +2,12 @@
 
 import java.util.Random;
 import java.util.concurrent.ForkJoinPool;
-import java.util.concurrent.RecursiveTask;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Callable;
 
 @SuppressWarnings("serial")
-public class Sum extends RecursiveTask<Integer> {
+public class Sum implements Callable<Integer> {
```

Comment:
When refactoring ForkJoinPools, I think we'd also want to refactor the ForkJoinTasks. Since this one computed and returned a result, I change it to implementing Callable.